benchmarking/runs/real_data/cosc_341/scenario1.py

```diff
@@ -0,0 +1,231 @@
+import math
+import os
+from typing import Dict
+
+import typer
+
+from api.ai.interfaces.algorithm_config import (
+    PriorityAlgorithmConfig,
+    WeightAlgorithmConfig,
+    RandomAlgorithmConfig,
+    GroupMatcherAlgorithmConfig,
+)
+from api.dataclasses.enums import ScenarioAttribute, Gender, AlgorithmType
+from benchmarking.data.real_data.cosc341_w2021_t2_provider.providers import (
+    COSC341W2021T2AnsweredSurveysStudentProvider,
+)
+from benchmarking.evaluations.graphing.graph_metadata import GraphData
+from benchmarking.evaluations.metrics.average_solo_status import AverageSoloStatus
+from benchmarking.evaluations.metrics.average_timeslot_coverage import (
+    AverageTimeslotCoverage,
+)
+from benchmarking.evaluations.metrics.cosine_similarity import (
+    AverageCosineDifference,
+)
+from benchmarking.evaluations.metrics.priority_satisfaction import PrioritySatisfaction
+from benchmarking.evaluations.scenarios.cosc341.concentrate_timeslot_diversify_gender_min_2_and_diversify_year_level import (
+    ConcentrateTimeSlotDiversifyGenderMin2AndDiversifyYearLevel,
+    DiversifyGenderMin2ConcentrateTimeSlotAndDiversifyYearLevel,
+)
+from benchmarking.runs.interfaces import Run
+from benchmarking.simulation.goal_to_priority import goals_to_priorities
+from benchmarking.simulation.insight import InsightOutput, Insight
+from benchmarking.simulation.simulation_set import SimulationSet
+from benchmarking.simulation.simulation_settings import SimulationSettings
+
+
+class Scenario1(Run):
```

Comment:
nit;

it was a bad idea to name these runs "scenario", when we already have another type of object in this repo called Scenario, not back breaking, but it's icky

Reply:
I think I am going to leave it because it corresponds to something in the paper, but I can leave a comment to explain.

Reply:
Or I can rename it if you prefer and leave a comment to explain what scenario it corresponds to.